src/index.html

```diff
@@ -19,9 +19,9 @@
         <div class="sidebars" ng-controller="SidebarCtrl as sidebarCtrl">
             <div class="search" ng-controller="SearchCtrl as searchCtrl" ng-click="sidebarCtrl.searchClick()" ng-style="{'width': sidebarCtrl.searchWidth()}">
                 <div class="row sidetab-top">
-                    <div class="glyphicon glyphicon-search top-icon" ng-click="searchCtrl.submit()"></div>
+                    <div class="glyphicon glyphicon-search top-icon"></div>
                     <div class="input-group sidetab" ng-show="sidebarCtrl.showSearches()">
-                        <form class="form-horizontal" ng-submit="searchCtrl.submit()">
+                        <form class="form-horizontal">
```

Comment:
You never need to press enter in that field -- the query is watched and the search results updated on typing. That allowed the submit method to be made private too.